packages/renderer/src/stores/kubernetes-contexts-state.ts

```diff
@@ -166,6 +166,31 @@ export const kubernetesCurrentContextNodesFiltered = derived(
   ([$searchPattern, $nodes]) => $nodes.filter(node => findMatchInLeaves(node, $searchPattern.toLowerCase())),
 );
 
+// Pods
+
+export const kubernetesCurrentContextPods = readable<KubernetesObject[]>([], set => {
+  window
+    .kubernetesRegisterGetCurrentContextResources('pods')
+    .then(value => set(value))
+    .catch((err: unknown) => console.log('Error registering Kubernetes pods', err));
+  window.events?.receive('kubernetes-current-context-pods-update', (value: unknown) => {
+    set(value as KubernetesObject[]);
+  });
+  return () => {
+    window
+      .kubernetesUnregisterGetCurrentContextResources('pods')
+      .catch((err: unknown) => console.log('Error unregistering Kubernetes pods', err));
```

Comment:
pure question: do we expect it's only a à.log` or it's an `.error`

Reply:
Ditto, same pattern from #9700.